wordcloud/wordcloud_cli.py

```diff
@@ -146,10 +140,16 @@ def parse_args(arguments):
     if args.color:
         color_func = wc.get_single_color_func(args.color)
 
-    args.collocations = not args.no_collocations
-
     args.color_func = color_func
-    return args
+
+    args = vars(args)
+    # remove arguments not directly mapped
+    _ = args.pop('colormask', None)
```

Comment:
so these are just ignored? That seems odd.

Reply:
No - they are used to choose the right color function some lines above and mapped to the same argument for the word cloud instance.

Reply:
ah. maybe use ``args.pop('color')`` there so it's more obvious?

Reply:
Right. Just using the `vars(args)` part earlier.